internal/kafka/internal/config/dataplane_cluster_config.go

```diff
@@ -366,6 +432,19 @@ func readDataPlaneClusterConfig(file string) (ClusterList, error) {
 	}
 }
 
+func readOperatorsSubscriptionConfigFile(file string, subscriptionConfig *operatorsv1alpha1.SubscriptionConfig) error {
+	fileContents, err := shared.ReadFile(file)
+	if err != nil {
+		return err
+	}
+
+	if err = k8sYaml.UnmarshalStrict([]byte(fileContents), &subscriptionConfig); err != nil {
```

Comment:
can we just return `return k8sYaml.UnmarshalStrict([]byte(fileContents), &subscriptionConfig)` that would return an error or nil

Reply:
You are totally right. Thanks

Reply:
done